src/ai/backend/client/cli/app.py

```diff
@@ -24,21 +27,32 @@ def __init__(self, api_session: AsyncSession,
                  session_id: str,
                  app_name: str,
                  protocol: str,
+                 args: MutableMapping[str, str], 
```

Comment:
For flag-style arguments, we should allow `None` value to distinguish with empty string arguments.
Examples:
 * `-v` for the verbose flag without any args (as json: `{"-v": null}`)
* `-t` for a tag string which may be an empty string (as json: `{"-t": ""}`)

```suggestion
                 args: MutableMapping[str, Optional[str]], 
```